spec/features/user/user_can_add_movies_to_cart_spec.rb

```diff
@@ -4,7 +4,7 @@
   context "user visits movies index" do
     it "should be able to add movie to cart" do
       director = Director.create!(name: "Someone!")
-      movie = director.movies.create!(title: "Shawshank Redemption", description: "Tim Robbins in jail")
+      movie = director.movies.create!(title: "Shawshank Redemption", description: "Tim Robbins in jail", rating: 5)
```

Comment:
ain't it a pain to have to go change all these other tests because you've made the rating a required field when we didn't ask for it to be required? ;)